requirements/requirements.txt

```diff
@@ -1,6 +1,6 @@
 # Framework must-haves
 numpy>=1.16.4
-torch>=1.1.0
+torch>=1.5.0
```

Comment:
could you please try 1.3? It's our maximum minimal requirement during team discussions :)

Reply:
@Scitator I've tried and it didn't work. I'll make another fix